spring/spring-web-reactive/src/test/java/io/quarkus/ts/spring/web/reactive/bootstrap/SpringWebQuteReactiveIT.java

```diff
@@ -1,22 +1,16 @@
-package io.quarkus.ts.spring.web.reactive.reactive.boostrap;
-
-import static io.quarkus.ts.spring.web.reactive.reactive.MariaDBConstants.IMAGE_105;
-import static io.quarkus.ts.spring.web.reactive.reactive.MariaDBConstants.PORT;
-import static io.quarkus.ts.spring.web.reactive.reactive.MariaDBConstants.START_LOG;
-
-import org.junit.jupiter.api.condition.EnabledIfSystemProperty;
+package io.quarkus.ts.spring.web.reactive.bootstrap;
 
 import io.quarkus.test.bootstrap.MariaDbService;
 import io.quarkus.test.bootstrap.RestService;
 import io.quarkus.test.scenarios.QuarkusScenario;
 import io.quarkus.test.services.Container;
 import io.quarkus.test.services.QuarkusApplication;
+import io.quarkus.ts.spring.web.reactive.MariaDBConstants;
 
 @QuarkusScenario
-@EnabledIfSystemProperty(named = "ts.redhat.registry.enabled", matches = "true")
 public class SpringWebQuteReactiveIT extends AbstractSpringWebQuteReactiveIT {
 
-    @Container(image = IMAGE_105, port = PORT, expectedLog = START_LOG)
+    @Container(image = MariaDBConstants.IMAGE_10, port = MariaDBConstants.PORT, expectedLog = MariaDBConstants.START_LOG_10)
```

Comment:
105->10?

Reply:
Original file has different change: https://github.com/quarkus-qe/quarkus-test-suite/pull/1155/files#diff-1bc7fd6657bd738a1e60e2d8d0e9cc945fd91e89644cbd1d345afc89a730b09f

Reply:
Original file you pointed at https://github.com/quarkus-qe/quarkus-test-suite/pull/1155/files#diff-1bc7fd6657bd738a1e60e2d8d0e9cc945fd91e89644cbd1d345afc89a730b09f has also 105 -> 10. I believe it is related to the point that we now run this test even when Red Hat registry is not related, as 105 is only available there.

TBH @fedinskiy I didn't do the original chages in #1155 so I can't tell if they were right, but I can tell what this PR does is same as we have on main right now

https://github.com/quarkus-qe/quarkus-test-suite/blob/main/spring/spring-web-reactive/src/test/java/io/quarkus/ts/spring/web/reactive/bootstrap/SpringWebRestReactiveIT.java#L13

and test is passing, therefore I don't see a problem. Please give me more hint if you can see a problem, thanks

Reply:
Although I seem to review that PR, completely forgot.

Reply:
I recall https://github.com/quarkus-qe/quarkus-test-suite/pull/1152